app/views/Layout.scala.html

```diff
@@ -105,7 +105,7 @@
                         signOutUrl = Some(controllers.routes.LogoutController.logout.url),
                         accessibilityStatementUrl = Some("/accessibility-statement/customs-financials")
                     ),
-        backLink = Some(BackLink(href = backLink.getOrElse(emptyString))),
+        backLink = backLink.filter(_.nonEmpty).map(BackLink(_)),
```

Comment:
This looks good. I am actually confused as to where this empty string was coming from though?

Reply:
The empty string was coming from the previous implementation where `getOrElse(emptyString)` was used as a fallback, now we ensure that only non-empty and valid backLink values are wrapped in a BackLink object, removing the ambiguity caused by the empty string fallback.